src/elements/core/base-elements/button-base-element.ts

```diff
@@ -128,6 +146,25 @@ abstract class SbbButtonBaseElement extends SbbFormAssociatedMixin(SbbActionBase
     }
   };
 
+  private _formKeyPress = (event: KeyboardEvent): void => {
+    const supportedSubmitElementNames = ['input', 'sbb-date-input'];
+    const form = this.form;
+    if (
+      this.type === 'submit' &&
+      form &&
+      (event.key === 'Enter' || event.key === '\n') &&
+      supportedSubmitElementNames.includes((event.target as HTMLElement)?.localName)
+    ) {
+      // We prevent submitting twice because we control it ourselves
+      // We also prevent when disabled, otherwise it would trigger a submit
+      event.preventDefault();
```

Comment:
@kyubisation What would the possible implications be, when doing that? I feel it could be too invasive, but so far I could only solve the problem with that